atlasdb-impl-shared/src/main/java/com/palantir/atlasdb/transaction/impl/SnapshotTransaction.java

```diff
@@ -959,6 +959,17 @@ private int estimateSize(List<RowResult<Value>> rangeRows) {
             }
         }
 
+        if (AtlasDbConstants.hiddenTables.contains(tableRef)) {
```

Comment:
Cool, makes sense. So, the action should be to add `checkGetPreconditions` to `getRowsColumRange` as it has a couple of other checks that should be done.